CLEO4/CCustomOpcodeSystem.cpp

```diff
@@ -389,6 +389,13 @@ namespace CLEO
 		{
 			// process parameter as a pointer to string
 			GetScriptParams(thread, 1);
+
+			if (buf != nullptr)
```

Comment:
What does this do?

Reply:
Copy the string sent (via pointer) to the opcode to buffer (the "fmt" variable in some CLEO's string related opcodes functions).

CLEO do it to string variables. (Look some lines bellow)

Reply:
Ah, I see what it does now.